README.md

```diff
@@ -98,42 +63,203 @@ chmod +x '[path-to]/cortex/cortex-js/dist/src/command.js'
+### `TensorRT-LLM`
+| Model ID         | Variant (Branch)              | Model size        | CLI command                        |
+|------------------|-------------------------------|-------------------|------------------------------------|
+| llama3           | 8b-tensorrt-llm-windows-ampere       | 8B                | `cortex run llama3:8b-tensorrt-llm-windows-ampere`   |
+| llama3           | 8b-tensorrt-llm-linux-ampere     | 8B                | `cortex run llama3:8b-tensorrt-llm-linux-ampere` |
+| llama3           | 8b-tensorrt-llm-linux-ada   | 8B                | `cortex run llama3:8b-tensorrt-llm-linux-ada`|
+| llama3           | 8b-tensorrt-llm-windows-ada       | 8B                | `cortex run llama3:8b-tensorrt-llm-windows-ada`   |
+| mistral          | 7b-tensorrt-llm-linux-ampere     | 7B                | `cortex run mistral:7b-tensorrt-llm-linux-ampere`|
+| mistral          | 7b-tensorrt-llm-windows-ampere       | 7B                | `cortex run mistral:7b-tensorrt-llm-windows-ampere`  |
+| mistral          | 7b-tensorrt-llm-linux-ada   | 7B                | `cortex run mistral:7b-tensorrt-llm-linux-ada`|
+| mistral          | 7b-tensorrt-llm-windows-ada       | 7B                | `cortex run mistral:7b-tensorrt-llm-windows-ada`  |
+| openhermes-2.5   | 7b-tensorrt-llm-windows-ampere       | 7B                | `cortex run openhermes-2.5:7b-tensorrt-llm-windows-ampere`|
+| openhermes-2.5   | 7b-tensorrt-llm-windows-ada     | 7B                | `cortex run openhermes-2.5:7b-tensorrt-llm-windows-ada`|
+| openhermes-2.5   | 7b-tensorrt-llm-linux-ada   | 7B                | `cortex run openhermes-2.5:7b-tensorrt-llm-linux-ada`|
+
+> **Note**:
+> You should have at least 8 GB of RAM available to run the 7B models, 16 GB to run the 14B models, and 32 GB to run the 32B models.
+
 ## Cortex CLI Commands
+> **Note**:
+> For a more detailed CLI Reference documentation, please see [here](https://cortex.so/docs/cli).
+### Start Cortex Server
+```bash
+cortex 
+```
+### Chat with a Model
+```bash
+cortex chat [options] [model_id] [message]
+```
+### Embeddings
+```bash
+cortex embeddings [options] [model_id] [message]
+```
+### Pull a Model
+```bash
+cortex pull <model_id>
+```
+> This command can also pulls Hugging Face's models.
+### Download and Start a Model
+```bash
+cortex run [options] [model_id]:[engine]
+```
+### Get a Model Details
+```bash
+cortex models get <model_id>
+```
+### List Models
+```bash
+cortex models list [options]
+```
+### Remove a Model
+```bash
+cortex models remove <model_id>
+```
+### Start a Model
+```bash
+cortex models start [model_id]
+```
+### Stop a Model
+```bash
+cortex models stop <model_id>
+```
+### Update a Model Config
+```bash
+cortex models update [options] <model_id>
+```
+### Get an Engine Details
+```bash
+cortex engines <name> get
+```
+### Initialize an Engine
+```bash
+cortex engines <name> init [options]
```

Comment:
I think we changed `init` to `install`

Reply:
@vansangpfiev Updated. Can you please merge? I don't have permission to merge in this repo